main.go

```diff
@@ -128,6 +128,15 @@ var fixtures = Fixtures{
 		},
 		CurrentRevision: 0,
 	},
+	{
+		Name: "review-pr-from-fork",
```

Comment:
Maybe something shorter?
```suggestion
		Name: "pr-from-fork",
```

Reply:
[done](https://github.com/src-d/lookout-test-fixtures/pull/27/files#diff-7ddfb3e035b42cd70649cc33393fe32cR132), many thanks!